http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -716,6 +727,16 @@ public void setHttp2Configuration(Http2Configuration http2Configuration) {
             http2Configuration(http2Configuration);
         }
 
+        @Override
+        public Builder useNonBlockingDnsResolver(boolean useNonBlockingDnsResolver) {
+            this.useNonBlockingDnsResolver = useNonBlockingDnsResolver;
+            return this;
+        }
+
+        public void setUseNonBlockingDnsResolver(boolean useNonBlockingDnsResolver) {
+            useNonBlockingDnsResolver(useNonBlockingDnsResolver);
+        }
```

Comment:
Nit: we use boxed types (`Boolean`) for configuration, so that we can differentiate between `null` ("use default") and `false` (disable). It's useful if we ever need to change the defaults in the future.

Reply:
Done.